local_planner/src/nodes/histogram.h

```diff
@@ -31,24 +34,18 @@ class Histogram {
   Histogram(const int res);
   ~Histogram();
 
-  inline double get_bin(int x, int y) const {
-    wrapIndex(x, y);
-    return bin[x][y];
-  }
-
   inline double get_age(int x, int y) const {
     wrapIndex(x, y);
-    return age[x][y];
+    return age(x, y);
   }
 
   inline double get_dist(int x, int y) const {
     wrapIndex(x, y);
-    return dist[x][y];
+    return dist(x, y);
   }
 
-  inline void set_bin(int x, int y, double value) { bin[x][y] = value; }
-  inline void set_age(int x, int y, double value) { age[x][y] = value; }
-  inline void set_dist(int x, int y, double value) { dist[x][y] = value; }
+  inline void set_age(int x, int y, double value) { age(x, y) = value; }
+  inline void set_dist(int x, int y, double value) { dist(x, y) = value; }
 
   void upsample();
```

Comment:
Maybe a brief comment that explains what `upsample` and `downsample` do. It's non-obvious IMHO, if I understand correctly they always overwrite the current histogram, up/downsample by a a factor of 2. But the user is not allowed to call this function twice. These are crucial assumptions (actually kinda dangerous because they are not enforced anywhere, the code would just segfault if used illegally) so they need to be either well-documented or checked against